open-company/open-company-web
# Fix trial expired banner

Found an issue in expanded post when the trial expired banner is shown.

To test:
- signup with a new org
- go to Stripe and switch to test data
- find the new customer
- expire the trial of his current subscription
- go back in carrot
- do you see the trial expired banner?  (you might need to wait some time to see the banner)
- stream looks good on desktop?
- expanded post looks good on desktop?
- stream looks good on mobile?
- expanded post looks good on mobile?
